conbench/static/app.css

```diff
@@ -109,3 +109,10 @@ table.dataTable td {
   white-space: nowrap;
   overflow: hidden;
 }
+
+.table-entry {
```

Comment:
Why did we need to add this?

Reply:
Added my thoughts below, short reasoning: overwriting the existing table class